File: Backend/mapping-kotlin/src/main/kotlin/com/mhp/coding/challenges/mapping/models/db/Article.kt
package com.mhp.coding.challenges.mapping.models.db

import com.mhp.coding.challenges.mapping.models.db.blocks.ArticleBlock
import java.util.*

class Article(
    var title: String,
    var description: String? = null,
    var author: String? = null,
    var blocks: Set<ArticleBlock>,
    override var id: Long,
    override var lastModified: Date,
    override var lastModifiedBy: String? = null
) : DBEntity

File: Backend/mapping-dotnet/MHP.CodingChallenge.Backend.Mapping.Data/DB/Article.cs
using System;
using System.Collections.Generic;
using MHP.CodingChallenge.Backend.Mapping.Data.DB.Blocks;

namespace MHP.CodingChallenge.Backend.Mapping.Data.DB
{
    public class Article : DbEntity
    {
        public String Title { get; set; }

        public String Description { get; set; }

        public String Author { get; set; }

        public HashSet<ArticleBlock> Blocks { get; set; }
    }
}
